sdm/src/main/java/com/sap/cds/sdm/constants/SDMConstants.java

```diff
@@ -5,10 +5,17 @@ private SDMConstants() {
     // Doesn't do anything
   }
 
-  public static final String REPOSITORY_ID = System.getenv("REPOSITORY_ID");
+  public static final String REPOSITORY_ID = "da0d8a9f-34da-4d6b-b52d-5cac7dc5d139";
   public static final String BEARER_TOKEN = "Bearer ";
 
   public static final String TENANT_ID = "X-zid";
   public static final String DUPLICATE_FILES_ERROR =
       "The following files %s already present.Remove the file from draft or Rename the attachment to continue.";
+  public static final String ERROR_RENAMING_ATTACHMENT = "Error renaming attachment";
+  public static final String DUPLICATE_FILE_IN_DRAFT_ERROR_MESSAGE =
+      "The file(s) %s have been added multiple times. Please rename and try again.";
+  public static final String FILES_RENAME_WARNING_MESSAGE =
+      "The following files could not be renamed as they already exist:\n%s\n";
+  public static final String COULD_NOT_RENAME_THE_DOCUMENT = "Could not rename the document";
+  public static final String DOCUMENT_NOT_FOUND = "Document not found";
```

Comment:
when is this shown can it be more proper message even for Could not rename the document

Reply:
Updated these messages